packages/eslint-plugin/rules/jsdoc-requires-since.js

```diff
@@ -96,11 +108,13 @@ module.exports = iterateJsdoc(
 		}
 
 		sinceTags.forEach( ( tag, index ) => {
-			const [ versionString ] = tag.description.split( ' ', 1 );
-			const description = tag.description.slice( versionString.length );
+			const tagDescription = getTagDescription( tag );
```

Comment:
Updating `eslint-plugin-jsdoc` resulted in leading whitespace being trimmed from `tag.description`, which stopped the rule detecting unwanted whitespace between `@since` and the version number, and caused a test to fail.

I've added the `getTagDescription()` helper to fix this.